docs/04-_PalindromDOM.md

```diff
@@ -55,22 +55,30 @@ Property    | Type          | Default    | Description
 ```javascript
 palindrom.method()
 ```
-Attribute   | Type          | Description
+Attribute   | Arguments          | Description
 ---         | ---           | ---
-`unlisten`  | *HTMLElement* | Stop listening to DOM events
-`listen`    | *HTMLElement* | Start listening to DOM events
+`unlisten`  | None | Stop listening to DOM events
+`listen`    | *target: HTMLElement* | Start listening to DOM events
+`async morphUrl`    | *url: String*    | Navigates to a URL and morphs the page accordingly
```

Comment:
Maybe replace the description with: `Navigates to a URL by making a request using 'async getPatchUsingHTTP' followed by a History API call`.

I think the `morphs the page accordingly` fragment should be omitted, because Palindrom does not have a concept of a page, does it?

Reply:
Good point